services/gr-css-reports/ssh_config

```diff
@@ -0,0 +1,15 @@
+Host gr-css
```

Comment:
Host *
    UseRoaming no

Not likely necessary, but it's what you're overwriting in the base image when you copy this over.